libs/shared/barista-definitions/jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
I guess we can remove this file as these type definitions will never contain tests